modules/ROOT/pages/provision-adapter-configure-service-mesh-CRD.adoc

```diff
@@ -11,6 +11,8 @@ The adapter acts as a policy enforcement component that fetches policies from An
 
 After you provision the adapter, you must set the `istio-injection` label on the namespace and redeploy all your existing applications in that namespace to ensure that the Envoy sidecar is injected within each pod in the Kubernetes Cluster.
 
+If you have installed Anypoint Service Mesh with Red Hat OpenShift, you must set the `istio-cni` network-attachment-definition on the namespace where you are running the applications.
```

Comment:
```suggestion
If you have installed Anypoint Service Mesh with Red Hat OpenShift, you must set the `istio-cni` network-attachment definition in your application namespace.
```
```suggestion
If you have installed Anypoint Service Mesh with Red Hat OpenShift, you must set the `istio-cni` network-attachment-definition on the namespace where you are running the applications.
```

Reply:
Hi @kbostrom, the type of resource the user need to create is `network-attachment-definition` with the name `istio-cni`, not `network-attachment`

Reply:
Left as is. Made no changes.